src/main/java/org/cryptomator/common/PropertiesPreprocessor.java

```diff
@@ -0,0 +1,66 @@
+package org.cryptomator.common;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.regex.Pattern;
+
+public class PropertiesPreprocessor {
+
+	private static final Logger LOG = LoggerFactory.getLogger(PropertiesPreprocessor.class);
+	private static final Pattern TEMPLATE = Pattern.compile("@\\{(\\w+)}");
```

Comment:
So we need to quote `{` but not `}`? That looks odd.

Reply:
Well, you can try it yourself and see if you get it working 😉